librz/core/cmd_analysis.c

```diff
@@ -4805,6 +4805,79 @@ static void __analysis_esil_function(RzCore *core, ut64 addr) {
 	rz_analysis_esil_free(core->analysis->esil);
 }
 
+static void cmd_rzil_mem(RzCore *core, const char *input) {
+	switch (*input) {
+	case '+':
+	case '-':
+		eprintf("[WIP] Add Mem or Remove Mem\n");
+		break;
+	case '\0':
+		eprintf("[DEAD] MOVE TO RZIL_INIT\n");
+		// rz_core_analysis_rzil_init_mem(core);
+		// core->analysis->rzil->init_mem = true;
+		break;
+	default:
+		eprintf("Usage: aeim [addr] [size] [name] - initialize ESIL VM stack\n");
+		eprintf("Default: 0x100000 0xf0000\n");
+		eprintf("See ae? for more help\n");
+		return;
+	}
+}
+
+static void cmd_analysis_rzil(RzCore *core, const char *input) {
+	char *n;
+	int repeat_times;
+
+	switch (input[0]) {
+	case 'r': // "aezr"
+		// 'aer' is an alias for 'ar'
+		// cmd_analysis_reg(core, input + 1);
+		eprintf("[WIP] Remove it or replace it ?");
+		break;
+	case 's': // "aezs"
+		switch (input[1]) {
+		case '?': // "aez?"
+			rz_core_cmd0(core, "ae?~aes");
```

Comment:
?? what is this?

Reply:
Since we use the old ESIL and a new one. I try to introduce a new command family for `RZIL`. Though it's used for developer now.